recipes/glibc/glibc.inc

```diff
@@ -43,6 +43,7 @@ do_configure_siteinfo() {
 }
 
 EXTRA_OECONF += "\
+	libc_cv_slibdir='/lib' \
```

Comment:
Hmm...  This looks very much like the multilib fixing we still have not agreed on...